prov/efa/src/rxr/rxr_cq.c

```diff
@@ -310,6 +310,9 @@ void rxr_cq_queue_rnr_pkt(struct rxr_ep *ep,
 #endif
 	dlist_insert_tail(&pkt_entry->entry, list);
 
+	// Remove other packets that were sent with the packet that encountered RNR
```

Comment:
Should be `/* ... */`

The comment is a little confusing. maybe explain what `pkt_entry->send_wr.wr.next` is